benchmarks/annotation_nquery.py

```diff
@@ -0,0 +1,196 @@
+"""Benchmarking for annotation storage nquery."""
+from numbers import Number
+from time import perf_counter
+from typing import Tuple
+
+import numpy as np
+from shapely import affinity
+from shapely.geometry import Polygon
+
+from tiatoolbox.annotation.storage import (
+    Annotation,
+    AnnotationStore,
+    DictionaryStore,
+    SQLiteStore,
+)
+
+
+# Copied from tests, refactor into a common location?
+def cell_polygon(
+    xy: Tuple[Number, Number],
+    n_points: int = 20,
+    radius: Number = 10,
+    noise: Number = 0.01,
+    eccentricity: Tuple[Number, Number] = (1, 3),
+    repeat_first: bool = True,
+    direction: str = "CCW",
+) -> Polygon:
+    """Generate a fake cell boundary polygon.
+
+    Cell boundaries are generated an ellipsoids with randomised eccentricity,
+    added noise, and a random rotation.
+
+    Args:
+        xy (tuple(int)): The x,y centre point to generate the cell boundary around.
```

Comment:
```suggestion
        xy (tuple(int)): 
            The x,y centre point to generate the cell boundary around.
```
Description on next line with an indent just to be consistent with the rest of the repo as you did before?